inc/osvr/Common/RegisteredStringMap.h

```diff
@@ -0,0 +1,159 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_RegisteredStringMap_h_GUID_066235BE_3687_44AD_C2A4_593D6E6780F3
+#define INCLUDED_RegisteredStringMap_h_GUID_066235BE_3687_44AD_C2A4_593D6E6780F3
+
+// Internal Includes
+#include <osvr/Common/Export.h>
+
+// Library/third-party includes
+#include <json/value.h>
+
+// Standard includes
+#include <string>
+#include <stdint.h>
+#include <vector>
+
+namespace osvr {
+namespace common {
+
+#define MAX_REGISTRY_SIZE 100
```

Comment:
Nooo! Big code guideline violation. (And use an enlargeable structure instead of having a fixed size)